packages/strapi-utils/lib/content-types.js

```diff
@@ -67,7 +67,12 @@ const getWritableAttributes = (model = {}) => {
 };
 
 const getNonVisibleAttributes = model => {
-  return _.uniq([model.primaryKey, ...getTimestamps(model), ...NON_VISIBLE_ATTRIBUTES]);
+  return _.uniq([
+    model.primaryKey,
+    ...NON_VISIBLE_ATTRIBUTES,
+    ...getTimestamps(model),
+    ...getNonWritableAttributes(model),
```

Comment:
Why this change ?

Reply:
the const is actually equal to the NON writable const + published at. In fact we need to add all the non writable attributes. Until no they were static so the const was enough. but now we have locale et localizations that are dynamic so I added the non writable attributes dynamically